pkgs/development/python-modules/python-emv/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub
+, click, enum-compat, pyscard, pycountry, terminaltables
+, pytestCheckHook, unittest2
+}:
+
+buildPythonPackage rec {
+  pname = "python-emv";
+  version = "1.0.9";
+
+  src = fetchFromGitHub {
+    owner = "russss";
+    repo = "python-emv";
+    rev = "v${version}";
+    hash = "sha256:1ldibcy1wgnaimf8ri79lg8im1mhfqajrigfflvizw0hb0ivh8al";
```

Comment:
```suggestion
    hash = "1ldibcy1wgnaimf8ri79lg8im1mhfqajrigfflvizw0hb0ivh8al";
```

Reply:
I prefer this syntax; it's easier to copy-and-paste out of the output from nix. I'd use the SRI syntax if my copy of nix was giving me that, but it isn't at the moment.